src/main/webapp/dev/js/common/feedbackResponseComments.es6

```diff
@@ -297,6 +320,12 @@ function hideResponseCommentAddForm(recipientIndex, giverIndex, qnIndex, section
         $(`#responseCommentTable${id}`).css('margin-top', '0');
         $(`#responseCommentTable${id}`).hide();
     }
+    const checkboxesInAddFormTable = getCheckBoxesInFormTable($(`#showResponseCommentAddForm${id} > form`));
```

Comment:
The code here (lines 323-328) and below (lines 415-422) seem rather repetitive. Is it possible to extract the behaviour into another function?